openapi.yml

```diff
@@ -5303,11 +5502,86 @@ components:
       - meta
       - meta_attr
       - package_versions
+      - references
       - resolution
       - state
       - title
       - trackers
       - uuid
+    FlawReference:
+      type: object
+      description: FlawReference serializer
+      properties:
+        description:
+          type: string
+        flaw:
+          type: string
+          format: uuid
+        type:
+          $ref: '#/components/schemas/Type154Enum'
+        url:
+          type: string
+          format: uri
+          maxLength: 2048
+        uuid:
+          type: string
+          format: uuid
+          readOnly: true
+        embargoed:
+          type: boolean
+          description: The embargoed boolean attribute is technically read-only as
+            it just indirectly modifies the ACLs but is mandatory as it controls the
+            access to the resource.
+        created_dt:
+          type: string
+          format: date-time
+          readOnly: true
+        updated_dt:
+          type: string
+          format: date-time
+          description: The updated_dt timestamp attribute is mandatory on update as
+            it is used to detect mit-air collisions.
+      required:
+      - created_dt
+      - embargoed
+      - flaw
+      - updated_dt
+      - url
+      - uuid
+    FlawReferencePost:
+      type: object
+      description: FlawReference serializer
+      properties:
+        description:
+          type: string
+        flaw:
```

Comment:
This should be rather in path like here https://github.com/RedHatProductSecurity/osidb/blob/master/openapi.yml#L4368